docs/using_inference/http_api.md

```diff
@@ -55,22 +55,22 @@ image = cv2.imread(image_file)
 
 #Configure client
 client = InferenceHTTPClient(
-    api_url="http://localhost:9001", # route for local inference server
+    api_url="https://detect.roboflow.com", # route for local inference server
```

Comment:
Why is this URL changing? We're showing how to use Inference on your local server in this guide.

Reply:
you have the localhost example above this in the docs, and it is convenient to copy this whole code snippet to visualize and have it work without having to deploy docker. imo we have too much localhost and not enough detect.roboflow.com